CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ All notable changes to this project will be documented in this file. The format
   - Fix units availability ([#1397](https://github.com/bloom-housing/bloom/issues/1397))
 
 - Added:
-
+  - Adds units to listings ([#1350](https://github.com/bloom-housing/bloom/issues/1350))
```

Comment:
The number here should be 1448 and it should probably be under the Frontend section

Reply:
Updated